test/unit/angularStorage/services/storeSpec.js

```diff
@@ -69,6 +69,100 @@ describe('angularStorage store', function() {
   }));
 
 });
+describe('angularStorage store: cookie fallback', function() {
+
+    /* these tests ensure that the cookie fallback works correctly.
+    *
+    * note - to confirm that cookiestore was used we attempt to retrieve the value from the cookie
+             since this bypasses our service, the result will not have been json parsed
+             therefore we use JSON.stringify on the expected value, so comparing like for like 
+    *
+    */
+    
+    var windowMock, $cookieStore;
+    
+    /* provide a mock for $window where localStorage is not defined */
+    beforeEach(module("angular-storage.store", function ($provide) {
```

Comment:
As we won't make angular-storage depend on `ngCookies`, what we should do is that in the tests, the module that gets created for testing should depend on `ngCookies` which is what would happen if we were doing a real app. Makes sense?